plugin/trino-hive/pom.xml

```diff
@@ -269,6 +269,24 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
airlift has also an http-client

```
        <dependency>
            <groupId>io.airlift</groupId>
            <artifactId>http-client</artifactId>
        </dependency>
```

Reply:
I looked into this and found that we cannot use this airlift http-client because the httpClient used by the `THttpClient` in the thrift library needs a HttpClient from Apache Http Components. There is a way to use the HttpUrlConnection based THttpClient but based on the documentation I see https://github.com/apache/thrift/blob/v0.18.1/lib/java/src/main/java/org/apache/thrift/transport/THttpClient.java, it performance issues under heavy load as well as the issue described in [THRIFT-970](https://issues.apache.org/jira/browse/THRIFT-970).

Reply:
cc @electrum